sysdig/resource_sysdig_monitor_alert_anomaly_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build tf_acc_sysdig
+//go:build tf_acc_sysdig || tf_acc_monitor
```

Comment:
alerts are not using v2 client, again wont work

Reply:
v1 is still alive and we are using it for multiple scenarios in our framework, so there is no problem at all.

Reply:
V1 is still alive but terraform provider does not support it. You cant use v1 alerts resources via terraform ATM.